cmd/snap-confine/mount-support.c

```diff
@@ -203,12 +203,34 @@ struct sc_mount_config {
 	const char *rootfs_dir;
 	// The struct is terminated with an entry with NULL path.
 	const struct sc_mount *mounts;
+	// Same as the structure above, but this is malloc-allocated.
+	struct sc_mount *dynamic_mounts;
 	sc_distro distro;
 	bool normal_mode;
 	const char *base_snap_name;
 	const char *snap_instance;
 };
 
+static void sc_create_mount_points(const char *scratch_dir,
+                                   const struct sc_mount *mounts)
+{
+	char dst[PATH_MAX] = { 0 };
+	sc_identity old = sc_set_effective_identity(sc_root_group_identity());
+	// Bind mount certain directories from the host filesystem to the scratch
```

Comment:
this comment is not super clear, it sounds like the code here does the bind mounts but actually is just creating the target for those. Maybe we need to reformulate.

Reply:
Yea you are right, the comment does not match the code. I'll rewrite the docs for this func